### ISSUE TITLE
Additional healer options?

### ISSUE THREAD

**Issue by bobosmith01** [lushen124/Universal-FE-Randomizer#169]
I know there's an option to leave the healers, but could we have an option to make key healers (Claude so you can survive that one time, and Ethlin/Lana/Mana just so you have a healer) instead of all healers? It felt odd to make all of them neccesarily healers, but now that I'm in Ch 4 with no healers whatsoever? Yikes.

**Comment by rambon99** [comment]
Yeah! Something like a guaranteed dedicated healer (that comes with a staff, I had one session where my only unit that could uses staves was sigurd as a Master Knight, but he didn't come with any staves so no healing in prologue) in prologue and claude squad would be nice. I wouldn't necessarily want it to always be ethlyn/claude, cus part of the fun of randomizers is having characters be different classes, but having at least one of the characters in that chapter be guaranteed healers would be awesome. Other games without healers are generally fine because of vulnearies, but in this one is brutal because the only other way to heal your units is with churches.

**Comment by lushen124** [comment]
The retain Healers should be what you're looking for, but I only have it affecting Edain, Claud, Lana/Muirne, and Coirpre/Charlot, giving you at least 1 healer by Chapter 1 at the latest in Gen 1 and immediately in Gen 2.

Ethlyn, Lachesis, Deirdre, Julia, and Nanna/Jeanne are not included in this list since the list of healing capable, unpromoted classes is very small (essentially Troubadour, Priest/Cleric, Princess, and Light Priestess) and that's a decent chunk of characters that would not be randomized with the option on.

Perhaps a control to set the number of staff users per generation would be better in this regard, but it'd also be hard to control when that healer shows up, so it's possible your healers come at the very end of the generation. It'd certainly require a bit more thought to implement, but I'm open to the idea.
